eZ/Publish/Core/Repository/Mapper/ContentDomainMapper.php

```diff
@@ -119,6 +127,19 @@ public function buildContentDomainObject(
 
         $versionInfo = $this->buildVersionInfoDomainObject($spiContent->versionInfo, $prioritizedLanguages);
 
+        $contentInfo = $versionInfo->getContentInfo();
+        $mainLocation = $contentInfo->getMainLocation();
+
+        // For performance reasons 'countLocationsByContent' is moved to if
+        if ($mainLocation === null && $this->locationHandler->countLocationsByContent($contentInfo->id) > 0) {
+            $this->logger->error(
+                sprintf(
+                    'Main location for content of ID = %d doesn\'t exist yet this content has locations assigned.',
+                    $contentInfo->id
```

Comment:
Isn't this an instance which already has `getId` method?

Reply:
Yes, it has after previous PR, resolved in https://github.com/ezsystems/ezplatform-kernel/pull/364/commits/528400a5ab0ec3242c00c64538fd48c669836323

Reply:
Well, mostly I meant this line, but fix at 134 is good too, missed it.
```suggestion
                    $contentInfo->getId()
```